tests/pytests/test_issues.py

```diff
@@ -97,6 +97,31 @@ def test_1667(env):
   env.expect('ft.search idx a').equal([0])
   env.expect('ft.search idx b').equal([1, 'doc_b', ['text', 'b']])
 
+@skip(cluster=False)
+def test_MOD_7454(env: Env):
+  env.cmd('FT.CREATE', 'idx', 'SCHEMA', 'n', 'NUMERIC')
+  n_docs = 1100 * env.shardsCount # We need more than 1000 docs in each shard
```

Comment:
Better to take this value from the config rather than hardcode the current value, such that if its default is ever changed this test will stay relevant

Reply:
Wasn't there a config-param for it? 🤔

Reply:
There is a parameter in the config structure, but no dedicated function to access it 😢

Reply:
Nope, than nothing to do here